doc/dev/vstart-runner.rst

```diff
@@ -0,0 +1,81 @@
+Running Python tests locally
+===============================
+
+Running your first test
+--------------------------
+Python tests in Ceph repo can be executed on the local machine using
+`vstart_runner.py`_. To do that you'd need `teuthology`_
+installed on your machine -
+
+::
+    $ git clone https://github.com/ceph/teuthology
+    $ cd teuthology/
+    $ virtualenv ./venv
+    $ source venv/bin/activate
+    $ pip install --upgrade pip
+    $ pip install -r requirements.rxt
+    $ python setup.py develop
+    $ deactivate
+
+Above steps installs teuthology in a virtual environment.
+
+*Note*: The pip command above is pip2, not pip3.
+
+*Note*: vstart_runner.py as well as most tests in qa/ are only compatible
+        with ``python2``. Therefore, use ``python2`` to run the tests locally.
+
+In a Ceph repository that has been successfully built perform following steps
```

Comment:
```suggestion
In a successfully built Ceph repository perform the following steps
```

Reply:
Not sure how this makes any difference.

Reply:
It's just a suggestion.

Reply:
Oh, ok. I thought grammar or syntax was wrong there.